src/migration/1644874601012-fiware-data-target.ts

```diff
@@ -0,0 +1,26 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
I have some issues with the database changes here.
With the new menu/page for picking data targets, we are opening up for multiple new data targets - which is a great new feature. 

This database change just expands the current data_target table with the FIWARE specific fields - this may become an issue later on where we will have to expand the model each time a new data target type is added. 

I would rather have a separate table or some other solution that doesn't fill out the table with fields that are only sometimes relevant.